guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -1,56 +1,68 @@
 [id="Resolving_Package_Dependencies_{context}"]
 = Resolving Package Dependencies
 
-In {Project}, you can use the package dependency resolution feature, also known as dependency solving, to ensure that any dependencies that packages have within a Content View are added to the dependent repository as part of the Content View publication process.
+{Project} can add dependencies of packages in a Content View to the dependent repository when publishing the Content View.
+To configure this, you can enable _dependency solving_.
 
-Dependency solving is relevant only in limited contexts.
-In the most common use cases, such as adding a security errata to a Content View with incremental update, dependency solving is likely unnecessary and can cause significant delays to Content View publication.
+For example, dependency solving is useful when you incrementally add a single package to a Content View version.
+You might need to enable dependency solving to install that package.
 
-The package dependency resolution feature does not take into account which packages are installed on your clients.
-Dependency solving considers only packages within the repositories of the Content View.
+However, dependency solving is unnecessary in most situations.
+For example:
+
+* When incrementally adding a security errata to a Content View, dependency solving can cause significant delays to Content View publication without major benefits.
+* In case of an outdated Content View version, the packages in a newer erratum might depend on packages outside that Content View.
```

Comment:
My original sentence doesn't make sense. Let's try something like:

"A newer erratum's packages might have dependencies that are incompatible with an older content view version's packages."

Regardless of the age of the content view, there shouldn't begin to be dependencies from repositories outside of the content view (unless the user  does something silly :) )

Reply:
Applied